docs/sources/administration/api-keys/create-api-key.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Create an API key in Grafana
+menuTitle: Create an API key
+description: 'How to create an API key in Grafana'
+weight: 50
+keywords:
+  - API keys
+  - Service accounts
+---
+
+# Create an API key in Grafana
+
+To manage your computed workload with a user that is not tied to a user specifically.
+
+For more information about API keys refer to [About API keys in Grafana]({{< relref "./about-api-keys.md">}}).
+
+For more information about creating API keys via the API, refer to [Create API key via API]({{< relref "../../http_api/create-api-tokens-for-org.md#how-to-create-a-new-organization-and-an-api-token">}}).
+
+Before you begin:
+
+**To create an API key:**
+
+- Ensure you have permission to create and edit API keys. For more information about permissions, refer to [About users and permissions]({{< relref "../manage-users-and-permissions/about-users-and-permissions.md#">}}).
+
+1. Sign in to Grafana, hover your cursor over **Configuration** (the gear icon), and click **API Keys**.
+1. Click **New API key**.
+1. Enter a unique name for the key.
+1. In the **Role** field, select one of the following access levels you want to assign to the key.
```

Comment:
Broken formatting
<img width="1107" alt="Screenshot 2022-04-07 at 12 45 24" src="https://user-images.githubusercontent.com/1861190/162182124-c795557b-75ac-4d8c-96c8-1d56b8a7e440.png">

Reply:
The bullet points should be aligned with the first character of the step above. My suggestions have added space - just two of them. Generate the docs after you accept the suggestion to ensure I've done it correctly.